### DISCUSSION TITLE
Should these Examples be updated to not give the Errors?

### DISCUSSION THREAD

**Discussion by 90PlusWorks** [OpenModelica/OpenModelica#11162] [Q&A]
This is an Example from : 

http://omwebbook.openmodelica.org/static/QuickTour/PlanarPendulum.html

Does anyone know how to fix these errors?

class Pendulum "Planar Pendulum"
   constant Real PI=3.141592653589793;
   parameter Real m=1, g=9.81, L=0.5;
   Real F;
   output Real x(start=0.5),y(start=0);
   output Real vx,vy;
equation
   m*der(vx)=-(x/L)*F;
   m*der(vy)=-(y/L)*F-m*g;
   der(x)=vx;
   der(y)=vy;
   x^2+y^2=L^2;
end Pendulum;
{Pendulum}

simulate(Pendulum, stopTime = 4)

record SimulationResult
    resultFile = "C:/Users/th/AppData/Local/Temp/OpenModelica/Pendulum_res.mat",
    simulationOptions = "startTime = 0.0, stopTime = 4.0, numberOfIntervals = 500, tolerance = 1e-06, method = 'dassl', fileNamePrefix = 'Pendulum', options = '', outputFormat = 'mat', variableFilter = '.*', cflags = '', simflags = ''",
    messages = "LOG_SUCCESS       | info    | The initialization finished successfully without homotopy method.
LOG_SUCCESS       | info    | The simulation finished successfully.
",
    timeFrontend = 0.0093921,
    timeBackend = 0.074431,
    timeSimCode = 0.0066074,
    timeTemplates = 0.0248296,
    timeCompile = 7.6837183,
    timeSimulation = 3.0473103,
    timeTotal = 10.8483506
end SimulationResult;
"[<interactive>:1:34-1:67:writable] Warning: Components are deprecated in class.
[<interactive>:1:70-1:102:writable] Warning: Components are deprecated in class.
[<interactive>:1:105-1:110:writable] Warning: Components are deprecated in class.
[<interactive>:1:113-1:147:writable] Warning: Components are deprecated in class.
[<interactive>:1:150-1:166:writable] Warning: Components are deprecated in class.
[<interactive>:1:178-1:195:writable] Warning: Equation sections are deprecated in class.
Warning: Assuming fixed start value for the following 2 variables:
         $STATESET2.x:VARIABLE(start = /*Real*/($STATESET2.A[1]) * $START.x + /*Real*/($STATESET2.A[2]) * $START.y fixed = true )  type: Real
         $STATESET1.x:VARIABLE(start = /*Real*/($STATESET1.A[1]) * $START.vy + /*Real*/($STATESET1.A[2]) * $START.vx fixed = true )  type: Real
"

**Comment by casella** [comment]
@90PlusWorks what environment are you using to get that output?

**Reply by 90PlusWorks** [reply]
I also have a Mint installation, is that preferred?

This was on OMShell 1.1 conneccted to OpenModelica v1.21.0(64bit) on Windows

OS Name:                   Microsoft Windows 10 Enterprise
OS Version:                10.0.19044 N/A Build 19044
System Manufacturer:       Dell Inc.
System Model:              Latitude 5520
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 140 Stepping 1 GenuineIntel ~1690 Mhz
BIOS Version:              Dell Inc. 1.24.3, 2022/10/27

**Reply by casella** [reply]
Could you please report exactly what you typed in OMShell?
